gui/src/components/logs/FormattedLogMessage.svelte

```diff
@@ -1,20 +1,51 @@
 <script lang="ts">
-  import { parseSpans } from './parseSpans';
+  import { parseSpans, Span } from './parseSpans';
 
   export let message: string = '';
 
   const spans = parseSpans(message);
+
+  const spanStyle = (span: Span): string => {
+    const styles: string[] = [];
+    const addStyle = (key: string, value: string) => {
+      styles.push(`${key}: ${value}`);
+    };
+    if (span.forground != null) {
+      addStyle('color', span.forground);
+    }
+    if (span.background != null) {
+      addStyle('background', span.background);
+    }
+    switch (span.style) {
+      case 'bold':
+        addStyle('font-weight', 'bold');
+        break;
+      case 'faint':
+        addStyle('font-weight', 'lighter');
+        break;
+      case 'italic':
+        addStyle('font-style', 'italic');
+        break;
+      case 'underline':
+        addStyle('text-decoration', 'underline');
+      case 'blink':
```

Comment:
What's the intent here?

Reply:
I think we'd have to do like `addClass('blink')`. I'll ping you on Slack & we can hack it up quick.

Reply:
fixed in 3152906